### DISCUSSION TITLE
EOSphere WAX Technical How To Format

### DISCUSSION THREAD

**Discussion by Rossco99** [worldwide-asset-exchange/docs#52] [General]
I have started converting and populating the wax-developer Github with our Medium Content.

@rakeden just a couple of questions:

- Where do you allocate a "nav_order" number
- Are we able to correct the spelling of our guild name in the directory tree from eossphere to eosphere?
- What should the direction be on keeping the article updated ie: software versions / new config etc.
- I wasn't able to submit a normal Question Issue it just 404'd
- Can you have a read of how I formatted "How to Set Up a Solid WAX Mainnet Node" if you are happy I'll roll with it like this

Cheers

**Comment by rakeden** [comment]
coming back this issue we need to bring you on par with how the new site works, so you can keep adding articles in the near future.
Have a first look at this wiki on how to collaborate. The source is located inside the dev branch until we merge into main and release. 

Let's keep this thread open
